tests/providers/amazon/aws/operators/test_datasync.py

```diff
@@ -415,29 +464,42 @@ def test_get_many_tasks(self, mock_get_conn):
 
         self.client.create_task(
             SourceLocationArn=self.source_location_arn,
-            DestinationLocationArn=self.destination_location_arn
+            DestinationLocationArn=self.destination_location_arn,
         )
 
         # Check how many tasks and locations we have
         tasks = self.client.list_tasks()
-        self.assertEqual(len(tasks['Tasks']), 2)
+        self.assertEqual(len(tasks["Tasks"]), 2)
         locations = self.client.list_locations()
-        self.assertEqual(len(locations['Locations']), 2)
+        self.assertEqual(len(locations["Locations"]), 2)
 
         # Execute the task
-        result = self.datasync.execute(None)
-        self.assertIsNotNone(result)
-
-        task_arns = result
-        self.assertIsNotNone(task_arns)
-        self.assertTrue(task_arns)
-        self.assertEqual(len(task_arns), 2)
+        with self.assertRaises(AirflowException):
+            self.datasync.execute(None)
 
         # Assert 0 additional task and 0 additional locations
         tasks = self.client.list_tasks()
-        self.assertEqual(len(tasks['Tasks']), 2)
+        self.assertEqual(len(tasks["Tasks"]), 2)
```

Comment:
NIT: should we split that to two tests?

Reply:
I think it should be fine as-is; if its needed in future we can always split them up later.